ContestModel/src/org/icpc/tools/contest/model/internal/account/BalloonContest.java

```diff
@@ -0,0 +1,73 @@
+package org.icpc.tools.contest.model.internal.account;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.icpc.tools.contest.model.IAccount;
+import org.icpc.tools.contest.model.IContestObject;
+import org.icpc.tools.contest.model.IContestObject.ContestType;
+import org.icpc.tools.contest.model.IJudgement;
+import org.icpc.tools.contest.model.ISubmission;
+
+public class BalloonContest extends PublicContest {
```

Comment:
Mayba also add a comment to this class as to what it filters on?

Reply:
Added a description, and noticed a simpler way to determine num solved. New commit pushed.